bot/cogs/memes_cog.py

```diff
@@ -26,6 +28,29 @@ async def bubblewrap(self, ctx):
 
         await ctx.send(msg)
 
+    @commands.command()
+    async def Waldo(self, ctx, size=max_waldo_grid_size):
+
+        max_waldo_line_size = 6
+        new_line_waldo_chance = 10
+        msg = ''
+        count = 0
+        place = random.randint(0,size)
+
+        for i in range(size+1):
+            if i == place:
+                msg += '||WALDO|| '
+                count += 1
+            else:
+                msg += '||MALDO|| '
```

Comment:
what are your thoughts on randomizing the first letter of the non waldo ones, like `Maldo` `HALDO` `NALDO` etc

Reply:
I tried.

Didn't work. Spacing was off. BALDO has a different length than WALDO which has a different length than everything other than MALDO

Reply:
If you knew the proper length, it'd be incredibly easy to see it

Reply:
ahh gotcha, makes sense